src/App.tsx

```diff
@@ -1,9 +1,19 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const getVisibleMovies = () => {
+    return moviesFromServer.filter(movie => movie
+      .title.toLowerCase().includes(query.toLocaleLowerCase())
+      || movie.description.toLowerCase().includes(query.toLocaleLowerCase()));
+  };
```

Comment:
```suggestion
  const getVisibleMovies = () => {
    return moviesFromServer.filter(movie => 
      stringContainsCaseInsensitive(movie.title, query)
      || stringContainsCaseInsensitive(movie.description, query)
  };
```
move part of the logic to the function

Reply:
done